code/main/media/mediaModule.brs

```diff
@@ -96,6 +96,14 @@ function _adb_MediaModule(configurationModule as object, edgeRequestQueue as obj
             mediaHit.tsObject = tsObject
             return mediaHit
         end sub,
+
+        dump: function() as object
+            return {
+                clientSessionId: m._sessionManager.getActiveClientSessionId(),
```

Comment:
why not dump the active session and get all the info.

Reply:
We can do that, but it could cause the performance issue. Roku only suggests passing small data objects between threads (task node thread and main UI thread). The integration framework requires passing dump data per second, so let's only pass the required data for now.

Reply:
can we reuse getActiveClientSessionId to test if there is an active session?

Reply:
No, it's not accurate. It's possible to have an active session, but the clientSessionId is still empty.